src/RestaurantButton.jsx

```diff
@@ -0,0 +1,23 @@
+import React, { useState } from "react";
+
+const RestaurantButton = props => {
+  const [orders, countOrder] = useState(0);
+  const incrementOrders = () => {
+    countOrder(orders + 1);
+  };
```

Comment:
👍  Great stuff with this functionality again. Be careful with the names the exercises are asking for, this code works and if the naming makes sense to you as you're writing it then happy days, but if you're using your own names then make sure they're semantic, i.e.
```
  const incrementOrders = () => {
    countOrder(orders + 1);
  };
```

incrementOrders is a good name (incrementOrder would probably be better because it's only incrementing a single order), but we're not counting anything with the countOrder function, have a think about naming convention for the state variables